fhttp/httprunner.go

```diff
@@ -201,6 +208,9 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 		_, _ = fmt.Fprintf(out, "Sockets used: %d (for perfect keepalive, would be %d)\n", total.SocketCount, r.Options().NumThreads)
 	}
 	_, _ = fmt.Fprintf(out, "Uniform: %t, Jitter: %t\n", total.Uniform, total.Jitter)
+	for k, v := range total.IPCount {
+		_, _ = fmt.Fprintf(out, "IP address %s usage count: %d\n", k, v)
```

Comment:
maybe a header line, something like
```
IP addresses distribution:
1.1.1.1: 23
1.1.1.2: 7
...
```
and sort either by count or ip

Reply:
updated